app/_src/gateway/kong-enterprise/provenance-verification.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Verify Build Provenance for Signed Kong Images
+badge: enterprise
+---
+
+Starting with {{site.ee_product_name}} 3.6.0.0, Build provenance for docker container images is produced, and can be verified using `cosign` / `slsa-verifier` with attestations published to a Docker Hub repository.
+
+This guide provides steps to verify build provenance for signed {{site.ee_product_name}} Docker container images in two different ways:
+
+* A minimal example, used to verify an image without leveraging any annotations
+* A complete example, leveraging optional annotations for increased trust
+
+For the minimal example, you only need Docker Manifest Digest and a GitHub repo name.
+
+{:.important .no-icon}
+> Docker manifest digest is required for build provenance verification. The manifest digest can be different from the platform specific image digest for a specific distribution.
+
+For the complete example, you need the same details as the minimal example, as well as any of the optional annotations you wish to verify:
+
+| Shorthand | Description | Example Value |
+|---|---|---|
+| `<repo>` | Github repository | `kong-ee` |
+| `<workflow name>` | Github workflow name | `Package & Release` |
+| `<worflow trigger>` | Github workflow Trigger | `workflow_dispatch` |
+
+Because Kong uses Github Actions to build and release, Kong also uses Github's OIDC identity to generate build provenance for container images, which is why many of these details are Github-related.
```

Comment:
```suggestion
| `<repo>` | GitHub repository | `kong-ee` |
| `<workflow name>` | GitHub workflow name | `Package & Release` |
| `<workflow trigger>` | GitHub workflow trigger | `workflow_dispatch` |

Because Kong uses GitHub Actions to build and release, Kong also uses GitHub's OIDC identity to generate build provenance for container images, which is why many of these details are GitHub-related.
```